inst/stan/functions/common_params.stan

```diff
@@ -0,0 +1,56 @@
+/**
+ * \page common_parameters_page Common Parameters
```

Comment:
Can we rename this page to make it clear what they are common parameters for? i.e delay_parameter-docs or something?

Reply:
Good point. I've renamed the page here b95533e047bfc3611ce37bac82cecfbca7555047.